.travis.yml

```diff
@@ -1,10 +1,8 @@
 language: node_js
 
 node_js:
-  - "9"
-  - "8"
-  - "6"
-  - "4"
+  - "12"
+  - "10"
```

Comment:
node 8 has been dropped by which dependancy?

Reply:
huh... will add it back.